smoke-tests/src/test/java/com/splunk/opentelemetry/SmokeTest.java

```diff
@@ -66,6 +66,10 @@ protected List<ResourceMapping> getExtraResources() {
     return List.of();
   }
 
+  protected boolean getAutodetectServiceName() {
```

Comment:
NIt: might have been spelled `shouldGetAutodetectServiceName()`?

Reply:
renamed to `shouldAutodetectServiceName`